src/tools/cursors/MouseCursor.js

```diff
@@ -0,0 +1,66 @@
+import toolColors from '../../stateManagement/toolColors.js';
+
+const mousePosition = '8 8';
+// TODO -> Change color based on usage?
+// TODO -> White is probably the second worst color to black for being displayed
+// over medical images.
+//
+
+const defaultOptions = {
+  iconSize: 16,
+  viewBox: {
+    x: 32,
+    y: 32,
+  },
+};
+
+export default class MouseCursor {
+  constructor(svgGroupString, options) {
+    this.svgGroupString = svgGroupString;
+    this.options = options;
+
+    const svgString = this._generateSVGString();
+
+    this.blob = new Blob([svgString], { type: 'image/svg+xml' });
```

Comment:
This prevents recalc, but it would be nice if we could force an update to set the blob to a version with an updated size/color.

Reply:
Yeah, I guess the svg generation is pretty light...

I wonder how bad it is to just implement something like this:

get blob() {
  return this._generateSVGString();
}

(or put the contents of _generateSVGString into the blob getter.)

It'd rebuild the svg element every time you switch your primary tool, but that's likely not too expensive, and it'd at least be dynamic to state changes

Reply:
Did this in latest commit.